File: Ejercicios/Java/Ejercicio2.java
import java.util.ArrayList;
public class Ejercicio2 {
    public static void main(String[] args) {
        ArrayList<Double> notas = new ArrayList<>();
        notas.add(9.5); notas.add(8.0); notas.add(7.5);
        notas.add(10.0); notas.add(6.0); notas.add(8.5);
        double suma = 0;
        for(Double n : notas) suma += n;
        System.out.println("Promedio total: " + (suma / notas.size()));
    }
}

File: Ejercicios/C++/Ejercicio2.cpp
#include <iostream>
#include <vector>
using namespace std;
int main() {
    vector<double> notas = {9.5, 8.0, 7.5, 10.0, 6.0, 8.5}; // usando initializer list
    double suma = 0;
    for(double n : notas) suma += n;
    cout << "Promedio total: " << (suma / notas.size()) << "\n";
    return 0;
}
